kokoro/linux-test/test.sh

```diff
@@ -13,7 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Test Build Script.
+# Linux Build Script.
+set -ex
 
-# Work in Progress
-echo "Placeholder for test"
+BUILD_ROOT=$PWD
+SRC=$PWD/github/gapid/
+
+# Get bazel
+curl -L -k -O -s https://github.com/bazelbuild/bazel/releases/download/0.25.1/bazel-0.25.1-installer-linux-x86_64.sh
+mkdir bazel
+bash bazel-0.25.1-installer-linux-x86_64.sh --prefix=$PWD/bazel
+
+# Get GCC 7
+sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+sudo apt-get -q update
+sudo apt-get -qy install gcc-7 g++-7
+export CC=/usr/bin/gcc-7
+
+cd $SRC
+BUILD_SHA=${KOKORO_GITHUB_COMMIT:-$KOKORO_GITHUB_PULL_REQUEST_COMMIT}
+
+echo $(date): Tests started.
+$BUILD_ROOT/bazel/bin/bazel \
+    --output_base="${TMP}/bazel_out" \
+    test tests -c opt --config symbols \
```

Comment:
Can you try to remove `-c opt` here to see if it speeds up the test build+run duration?
`-c opt` is for an optimized build, `-c dbg` for a debug build, and by default it is a "fastbuild" which should hopefully build faster.